internal/contour/cluster.go

```diff
@@ -25,9 +25,21 @@ import (
 
 	"github.com/envoyproxy/go-control-plane/envoy/api/v2/auth"
 	"github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
+	envoy_api_v2_core4 "github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
+	google_protobuf "github.com/gogo/protobuf/types"
+	ingressroutev1 "github.com/heptio/contour/apis/contour/v1beta1"
 	"github.com/heptio/contour/internal/dag"
 )
 
+const (
+	// Default healthcheck / lb algorithm values
+	hcTimeout            = int64(2)
```

Comment:
You can drop the cast here, leaving them as untyped numeric constants, the compiler will figure it out.